src/test/java/com/ipiecoles/java/java350/model/EmployeTest.java

```diff
@@ -1,64 +1,229 @@
+        Employe employe = new Employe("Doe", "John", matricule, LocalDate.now().minusYears(nbYearsAnciennete), Entreprise.SALAIRE_BASE, performance, tempsPartiel);
+
+        //When
+        Double prime = employe.getPrimeAnnuelle();
 
         //Then
-        Assertions.assertThat(nbAnnee).isEqualTo(0);
+        Assertions.assertEquals(primeAnnuelle, prime);
+
     }
 
+
+    //TEST METHODE augmenterSalaire
     @Test
-    public void testGetNombreAnneeAncienneteNmoins2(){
+    public void testAugmenterSalaireDefault() throws EmployeException {
         //Given
         Employe e = new Employe();
-        e.setDateEmbauche(LocalDate.now().minusYears(2));
 
         //When
-        Integer nbAnnee = e.getNombreAnneeAnciennete();
+        double augmenterSalaire = Math.round(e.getSalaire() * e.augmenterSalaire(1.05 ));
 
         //Then
-        Assertions.assertThat(nbAnnee).isEqualTo(2);
+        Assertions.assertEquals(1597, augmenterSalaire);
+    }
+
+    @Test
+    public void testAugmentationSalaireNull() throws EmployeException{
+        //Given
+        Employe e = new Employe();
+        e.setSalaire(null);
+
+        //when
+        //Mise en place d'une exception dans Employe suite test TDD
+        try {
+            double augmentationSalaire =  e.augmenterSalaire(1.3);
+            Assertions.fail("lance une exception");
+        } catch (Exception exceptionNull) {
+            //Then
+            Assertions.assertEquals("Le salaire ne peux être null",
+                    exceptionNull.getMessage());
+        }
     }
 
     @Test
-    public void testGetNombreAnneeAncienneteNplus2(){
+    public void testAugmenterSalaireDefinit() throws Exception {
         //Given
         Employe e = new Employe();
-        e.setDateEmbauche(LocalDate.now().plusYears(2));
+        //Test avec un salaire différent de celui par défault
+        e.setSalaire(1200.50);
 
+        //On utilise Math.round pour arrondir le résultat
         //When
-        Integer nbAnnee = e.getNombreAnneeAnciennete();
+        double augmenterSalaire = Math.round(e.getSalaire() * e.augmenterSalaire(1.1));
 
         //Then
-        Assertions.assertThat(nbAnnee).isEqualTo(0);
+        Assertions.assertEquals(1321 , augmenterSalaire);
+
     }
 
+    @Test
+    public void testAugmenterSalairePourcentageZero() throws EmployeException {
+        //Given
+        Employe e = new Employe();
 
+
+        //When
+
+        try {
+            double augmenterSalaire =e.augmenterSalaire(0);
+            Assertions.fail("Doit lancer un exception");
+        } catch (EmployeException e6) {
+            //Then
+            Assertions.assertEquals("Le pourcentage ne peux être égale à 0 !!",
+                    e6.getMessage());
+        }
+
+    }
+
+    //TEST NbRtt pour l'année 2019 avec l'appel à la classe LocalDate
+
+    @ParameterizedTest(name="jours de l'année{0}, jourFeriesOuvres{1}, weekend{2},NbRtt{4}")
+    @CsvSource({
+                    "365, 9, 104, 9"
```

Comment:
Il aurait fallu grouper tes tests dans un seul tests paramétré, c'est justement l'intérêt !
```suggestion
                    "365, 9, 104, 9"}, {"365, 7, 104, 11"
```